src/plugins/ResizeReorder/ReorderCell.js

```diff
@@ -121,25 +129,139 @@ class ReorderCell extends React.PureComponent {
     }
 
     return (
-      <div className={reorderClasses} style={style}>
-        <ReorderHandle
-          {...this.props}
-          translateCell={this.translateCell}
-          onColumnReorderStart={this.onColumnReorderStart}
-          touchEnabled={this.props.touchEnabled}
-          height={this.props.height}
-          isRTL={this.context.isRTL}
-          columnKey={this.props.columnKey}
-          left={this.props.left}
-          onColumnReorderEndCallback={this.onColumnReorderEnd}
-        />
+      <div className={reorderClasses} style={style} ref={this.cellRef}>
+        {this.renderReorderHandle()}
         {content}
       </div>
     );
   }
+
+  renderReorderHandle() {
```

Comment:
I removed [ReorderHandle.js](https://github.com/schrodinger/fixed-data-table-2/pull/640/files#diff-f7901ec4690659028d87c9f5e715fed209bae9a28866e7f2715212b2b65c6db7) and replaced it with the `renderReorderHandle` method here because all it did was render a small UX handle.